executor/prepared.go

```diff
@@ -273,46 +273,44 @@ func ResetStmtCtx(ctx sessionctx.Context, s ast.StmtNode) {
 		sc.MemTracker.SetActionOnExceed(&memory.LogOnExceed{})
 	}
 
+	// TODO: Many same bool variables here.
+	// We should set only two variables (
+	// IgnoreErr and StrictSQLMode) to avoid setting the same bool variables and
+	// pushing them down to TiKV as flags.
```

Comment:
does MySQL use the same two variables to control the behavior of error and warning?